frontend/src/app/login/page.tsx

```diff
@@ -0,0 +1,165 @@
+'use client';
+
+import { useState } from 'react';
+import { userClient } from '../api/userClient';
+import { useRouter } from 'next/navigation';
+import {
+  Alert,
+  Avatar,
+  Box,
+  Button,
+  Link,
+  TextField,
+  Typography,
+} from '@mui/material';
+import NextLink from 'next/link';
+import { LockOutlined } from '@mui/icons-material';
+import { AxiosError } from 'axios';
+
+export default function LoginPage() {
+  const router = useRouter();
+  const [username, setUserName] = useState('');
+  const [password, setPassword] = useState('');
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    console.log("➡️ handleSubmit lefutott", { username, password }); // 🔥
+    setErrorMessage('');
+
+    const loginRequest = {
+      username: username,
+      password: password,
+    };
+
+    try {
+      await userClient.login(loginRequest);
+      router.push('/');
+    } catch (error) {
+      const axiosError = error as AxiosError<{ message?: string }>;
+      if (axiosError.response) {
+        if (axiosError.response.status === 409) {
```

Comment:
Is there a difference between handling 409 and non-409 errors? At a first glance it seems like you just set an error message in both cases. Either something is missing, or you don't need an if here.